src/services/gateways/aws.ts

```diff
@@ -184,7 +186,7 @@ type AWSConfig = {
 export const IASQL_EC2_TAG_NAME = 'IaSQL_Name';
 
 export class AWS {
-  private ec2client: EC2Client
+  private ec2client: EC2
   private ecrClient: ECRClient
```

Comment:
when we move the AWS client usage to the mappers we should use the `EC2`/`ECR` objects instead of the `ECRClient`/`EC2Client`. They are more DRY / concise

Reply:
oh really?